docs/BulkExport.md

```diff
@@ -38,6 +38,20 @@ https://test-fhir-server/$export
 
 ## What is supported
 
-For more details on Bulk Export, see the [Azure API for FHIR Export Data page](https://docs.microsoft.com/en-us/azure/healthcare-apis/export-data).
+For more details on Bulk Export, see the [Azure API for FHIR Export Data page](https://docs.microsoft.com/en-us/azure/healthcare-apis/export-data). 
+
+In addition to the query parameters specified in the Azure API For FHIR documentation, users can also use the \_format in FHIR Server. \_format allows a user to select a format for the file structure that the export job creates. Different formats can be defined in the appSettings by combining constants, folder level breaks ('/'), and known tags. The tags will be replaced with data when the job is run. The three supported tags are: 
+* **resourcename**: replaces with the resource type being exported
+* **timestamp**: replaces with a timestamp of the job's queried time
+* **id**: replaces with the GUID of the export job)
+
+To use the format, you will need to set the following settings in the appSettings:
+
+| appSetting | Description | Example Value|
+|------------|-------------|--------------|
+| FhirServer:Operations:Export:Formats:0:Name | Name of the format you plan to call | TestFormat |
+| FhirServer:Operations:Export:Formats:0:Format | Defines the format | test/\<resourcename>/\<id>/\<timestamp> |
```

Comment:
If a user wants to define multiple formats they can change the number in the setting. Technically we provide default values for 0 and 1, so we might want to recommend customers start with 2.

Reply:
Sounds good. I changed this to a variable. Let me know if this looks okay!